src/scripts/index.js

```diff
@@ -0,0 +1,16 @@
+import addTodos from './todo.js';
+
+
+const container = document.getElementById('container');
+
+const time = () => {
```

Comment:
instead of setting attributes seperately, you can create the input in a single line if you wish:
const time = $(`<div id='time'>${new Date().toLocaleTimeString()}</div>`)